code/src/components/TasksList.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+
+
+import { CheckBox } from './CheckBox'
+import { RemoveButton } from './RemoveButton'
+import "./taskslist.css"
+
+
+export const TasksList = () => {
+  // TODO - fetch all tasks from the store
+  const allTasks = useSelector((store) => store.tasks)
+
+  return (
+    <div className="tasks-container">
+      {allTasks.items.map((task) => (
+        <div key={task.id} className="task">
+          <CheckBox task={task} />
+          <div className="task-detail">
+            <div className="text"> {task.text} </div>
```

Comment:
In answer to your question in the pull request about why the strikethrough css didn't work: If you want to write the css `.checked .text`, then you'd need to add the `checked` class to something which contains the .text element (such as the .task div on line 17 above. Or, a simpler way would be to add to the class on this line:

```
<div className={task.complete ? 'task checked' : 'task'}>{task.text}</div>
```

Then, in your css, you could select this text:

```css
.text.checked {
  text-decoration: line-through;
}
```